quoridorcore/src/main/java/com/harush/zitoon/quoridor/core/model/Utils/GamePersistenceServiceImpl.java

```diff
@@ -1,4 +1,4 @@
-package com.harush.zitoon.quoridor.core.model;
+package com.harush.zitoon.quoridor.core.model.Utils;
```

Comment:
A class named `Service` should be under package `service`, since that's what it is and not a `util`